mn/widget.js

```diff
@@ -0,0 +1,271 @@
+
+    'commons.close': 'Хаах', // close button
+
+    /**
+     * Tooltips and balloons on closed widget
+     */
+    'monitor.pageViewersTitle': 'Хуудсыг үзэж буй хүмүүс',
+    'monitor.siteViewersTitle': 'Сайтыг үзэж буй хүмүүс',
+    'monitor.chatBalloon': 'ЧААТ', // keep upper case, if applicable
+    'monitor.presenceBalloon': 'Холбогдсон хүмүүс',
+
+    /**
+     * Menu
+     */
+    'status-header.options': 'Сонголтууд',
+    'status-header.flushChat': 'Энэ чатыг цэвэрлэ',
+    'status-header.disableChat': 'Энэ чатыг хаах',
+    'status-header.enableChat': 'Энэ чатыг идэвхжүүлэх',
+    'status-header.help': 'Тусламж',
+    'status-header.issue': 'Алдааг мэдээлэх',
+    'status-header.terms': 'Үйлчилгээний нөхцөл',
+    'status-header.privacy': 'Нууцлалын бодлого',
+    'status-header.signout': 'Гарах',
+    'status-header.signin': 'Орох',
+    'status-header.editUserProfile': 'Профайл засварлах',
+
+    'status-header.notRememberedLogin': 'Сануулаагүй',
+    // means that the login status is temporary and will not be remembered next time
+
+    'status-header.editUserProfileModal': 'Дэлгэцийн нэр болон зургийг солихын тулд <a>Профайл засварлах</a> дээр дарна уу',
+    // keep <a> and </a> unchanged and surrounding the translation of "edit your profile"   
+
+    /**
+     * Tab labels and tooltips
+     */
+    'tab.chatTabLabel': 'ЧААТ', // max 8 characters; keep upper case, if applicable
+    'tab.chatTitlePage': 'Хуудасны зочидтой хийх рийл-тайм чаат',
+    'tab.chatTitleSite': 'Сайтын зочидтой хийх рийл-тайм чаат',
+
+    'tab.rankingTabLabel': 'ХУУДСУУД', // max 8 characters; keep upper case, if applicable
+    'tab.rankingTitle': 'Үзэж буй хуудсууд',
+
+    'tab.heatmapTabLabel': 'ЗУРАГ', // max 8 characters; keep upper case, if applicable
+    'tab.heatmapTitle': 'Хаанаас зочилсон',
+
+    /**
+     * Heatmap
+     */
+    'heatmap.pageSwitchLabel': 'ХУУДАС', // max 12 characters; keep upper case, if applicable
+    'heatmap.siteSwitchLabel': 'САЙТ', // max 12 characters; keep upper case, if applicable
+    'heatmap.empty': 'Дата байхгүй',
+
+    /**
+     * Ranking
+     */
+    'ranking.foregroundTitle': 'Зочид',
+    'ranking.activeTitle': 'Чаатад буй хүмүүс',
+    'ranking.foregroundSwitchLabel': 'УНШИГЧИД', // max 12 characters; keep upper case, if applicable
+    'ranking.activeSwitchLabel': 'ЧААТЛАГЧИД', // max 12 characters; keep upper case, if applicable
+    'ranking.empty': 'Дата байхгүй',
+    'ranking.currentPage': 'Одоогийн хуудас',
+
+    'ranking.note': 'Топ 10 хуудсыг харуулж байна',
+    // note added to the end to the ranking list to explain that only the top ten pages are shown
+
+    /**
+     * Chat
+     */
+    'chat.welcomeMessage': 'Бусад зочидтой шууд чат бич. Бичсэн чат нийтэд харагдах ба $N $A хугацааны дараа алга болно.',
+    'chat.welcomeMessageJWT': 'Бусад зочидтой шууд чат бич. Бичсэн чат нийтэд харагдах ба $N $A хугацааны дараа алга болно.',
```

Comment:
In this case we want to say that the chat COULD be public but it does not necessarily mean that it will be.